wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/settings/SettingsProximitySharingScreen.kt

```diff
@@ -0,0 +1,93 @@
+package com.android.identity_credential.wallet.ui.destination.settings
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.android.identity.document.DocumentStore
+import com.android.identity.util.Logger
+import com.android.identity_credential.wallet.R
+import com.android.identity_credential.wallet.SettingsModel
+import com.android.identity_credential.wallet.navigation.WalletDestination
+import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
+import com.android.identity_credential.wallet.ui.SettingSectionSubtitle
+import com.android.identity_credential.wallet.ui.SettingToggle
+
+private const val TAG = "SettingsProximitySharingScreen"
+
+@Composable
+fun SettingsProximitySharingScreen(
+    settingsModel: SettingsModel,
+    documentStore: DocumentStore,
+    onNavigate: (String) -> Unit
+) {
+    ScreenWithAppBarAndBackButton(
+        title = stringResource(R.string.settings_proximity_sharing_screen_title),
+        onBackButtonClick = { onNavigate(WalletDestination.PopBackStack.route) },
+        scrollable = false
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp)
+        ) {
+            SettingSectionSubtitle(title = stringResource(R.string.settings_proximity_sharing_screen_device_engagement_settings_title))
+            Text(
+                text = stringResource(R.string.settings_proximity_sharing_screen_device_engagement_settings_explanation),
+                style = MaterialTheme.typography.bodySmall,
+                color = MaterialTheme.colorScheme.onSurfaceVariant,
+                fontStyle = FontStyle.Italic,
+            )
+            SettingToggle(
+                title = stringResource(R.string.settings_proximity_sharing_screen_nfc_static_handover_title),
+                isChecked = settingsModel.nfcStaticHandoverEnabled.observeAsState(false).value,
+                onCheckedChange = { settingsModel.nfcStaticHandoverEnabled.value = it }
+            )
+            SettingToggle(
+                title = stringResource(R.string.settings_proximity_sharing_screen_nfc_negotiated_handover_title),
+                isChecked = !settingsModel.nfcStaticHandoverEnabled.observeAsState(false).value,
```

Comment:
Both this SettingToggle and the one above it are changing the same underlying nfcStaticHandoverEnabled value. Do we need both? Is this just to clarify that enabling one disables the other?

(and if so, do we have something like a radio button instead?)

Reply:
It's intentional that there are to toggles here.